backend/src/main/java/pt/ulisboa/tecnico/socialsoftware/tutor/user/User.java

```diff
@@ -466,5 +529,8 @@ public QuizAnswer getQuizAnswer(Quiz quiz) {
                 .orElse(null);
     }
 
-
+    public void removeFromCourseExecutions(){
```

Comment:
Better to have this inside a remove() method that does what is necessary to remove a user